src/main/java/au/gov/digitalhealth/terminology/amtflatfile/Amt2FlatFile.java

```diff
@@ -352,12 +362,29 @@ private void writeFlatFile(Path path) throws IOException {
                             + " for MPP " + mpp;
                 	logger.warning(message);
 
-//                    testSuite.addTestCase("Mismatch", message, "MPP mismatch (" + mpp.getId() + ")", "ERROR");
+                    testSuite.addTestCase("Mismatch", message, this.getClass().getName(), "MPP mismatch (" + mpp.getId() + ")", "ERROR");
+                }
+
+                if ( count % 10000 == 0) {
```

Comment:
So that there's more logging as it goes
```suggestion
                if ( count % 1000 == 0) {
```